ichthyology_importer.py

```diff
@@ -77,10 +77,8 @@ def get_catalog_number(self, filename):
         return f'{number}', collection
 
     def build_filename_map(self, full_path):
```

Comment:
full_path=full_path.lower() , some ubuntu systems are case sensitive thus this errored out in the test env. , I removed it and did a test import, but I might want to double check that taking this out doesn't throw a wrench in any functions down the line.